include/sdsl/int_vector_buffer.hpp

```diff
@@ -266,50 +243,56 @@ class int_vector_buffer
             return (m_buffersize*width())/8;
         }
 
-        //! Set the buffersize in bytes
-        void buffersize(uint64_t buffersize)
```

Comment:
Because this looks bad to me. If one wants to change the buffer size, just reinitialize the int_vector_buffer with a different value. That's equivalent.